### DISCUSSION TITLE
Open explorer with sbase

### DISCUSSION THREAD

**Discussion by michalfain** [seleniumbase/SeleniumBase#2811] [Q&A] [answered]
Hi,
Is there a way to open url with explorer, or with edge in explorer mode?

**Comment by mdmintz** [comment, marked as answer]
Use `browser="ie"`, or see https://github.com/seleniumbase/SeleniumBase/discussions/751#discussioncomment-639734.

**Comment by michalfain** [comment]
Thanks!
Where do I insert that line? I can't find the reference

**Reply by mdmintz** [reply]
It depends on which of the [23 Syntax Formats](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/syntax_formats.md) you're using.

Eg. `pytest --ie` / `Driver(browser="ie")` / `SB(browser="ie")`

**Reply by michalfain** [reply]
Ok, thanks!
